calour/_testing.py

```diff
@@ -62,6 +62,9 @@ def setUp(self):
         self.q2_cfs_map = join(test_data_dir, 'cfs-map.txt')
         self.q2_cfs_repseqs = join(test_data_dir, 'cfs-rep-seqs.qza')
         self.q2_cfs_taxonomy = join(test_data_dir, 'cfs-taxonomy.qza')
+        # an experiment used to create a RatioExperiment
```

Comment:
these are only used in ratio_exp? if so, they should be in the setUp() in test_ratio_experiment.py. Putting them here slows down all the tests.

Reply:
fixed